core/system_metric/sys_metric_stat.go

```diff
@@ -178,7 +180,11 @@ func retrieveAndUpdateCpuStat() {
 
 	cpuRatioGauge.Set(cpuPercent)
 
-	currentCpuUsage.Store(cpuPercent)
+	numcpu := runtime.NumCPU()
```

Comment:
why divide by the cpu num？

Reply:
"getProcessCpuStat" return multiple CPU cores usage percentage. That is to say, the value can also be > 100.0 in case of a process running multiple threads on different CPUs.

It divide by the cpu num to normalizate this value to [0, 1.0]. For the following reasons:

1. ease of use. The user should not be related to the cpu number of the machine.
2. compatibility to System Adaptive Protection threshold value. This threshold value should be in [0, 1.0] when rule type is CPUusage.